platform/darwin/src/MGLGeometry.h

```diff
@@ -173,6 +173,17 @@ NS_INLINE MGLCoordinateBounds MGLCoordinateBoundsOffset(MGLCoordinateBounds boun
     return offsetBounds;
 }
 
+/**
+ YES if the coordinate is valid or NO if it is not.
+ Considers extended coordinates.
+ */
+NS_INLINE BOOL MGLLocationCoordinate2DIsValid(CLLocationCoordinate2D coordinate) {
```

Comment:
Let’s take this function private. It’s confusing to have a function named so similarly to `CLLocationCoordinate2DIsValid()` yet behave so differently. Note that the behavior of allowing longitudes beyond ±180° is specific to this map SDK and shouldn’t be used, for instance, with code that targets the navigation SDK.